core/src/main/java/hudson/model/Queue.java

```diff
@@ -2317,6 +2353,71 @@ public CauseOfBlockage getCauseOfBlockage() {
     }
 
     /**
+     * {@link Item} in the {@link Queue#flyWeightTasks} stage.
+     */
+    public final class FlyWeightItem extends NotWaitingItem {
```

Comment:
The reason I was I wanted to track which flyweighttask was in the buildable list too, but then I ended up doing something else. I could probably just use the BlockedItem list type or one of the existing